content/en/docs/advanced-tasks/catalog-update-formulary.md

```diff
@@ -0,0 +1,305 @@
+---
+title: "Catalog Update Formulary"
+linkTitle: "Catalog Formulary"
+weight: 5
+---
+
+## Background
+[File-based-catalogs][file-based-catalog-spec] (FBC) and [catalog templates][templates-doc] (templates)  provide operator authors with standardized schemas to express operator upgrade graphs.  However, without explicit tooling users require clear examples of how to achieve routine goals.  This document is an attempt to establish a formulary of common operations, specificially with the intention of making these pieces automatable.  This is in no way an exhaustive list.
```

Comment:
Nit: Misspelling:
```suggestion
[File-based-catalogs][file-based-catalog-spec] (FBC) and [catalog templates][templates-doc] (templates)  provide operator authors with standardized schemas to express operator upgrade graphs.  However, without explicit tooling users require clear examples of how to achieve routine goals.  This document is an attempt to establish a formulary of common operations, specifically with the intention of making these pieces automatable.  This is in no way an exhaustive list.
```

Reply:
HAHAHAHA.  Confirmation bias is alive and well.  I never even saw it.